archetypes/Stake/FilterSelects/index.tsx

```diff
@@ -0,0 +1,105 @@
+                    </BaseFilters.Preview>
+                }
+                buttonClasses="action-button"
+            >
+                <BaseFilters.Content>
+                    <div>
+                        <BaseFilters.Heading>Market</BaseFilters.Heading>
+                        <BaseFilters.Dropdown
+                            variant="default"
+                            iconSize="xs"
+                            placeHolderIcon={
+                                Object.entries(MarketFilterEnum).find(
+                                    ([_key, val]) => val === state.marketFilter,
+                                )?.[0] as LogoTicker
+                            }
+                            value={state.marketFilter}
+                            options={MARKET_FILTER_OPTIONS}
+                            onSelect={onMarketSelect}
+                        />
+                    </div>
+                    <BaseFilters.Wrapper>
+                        <BaseFilters.DropdownContainer>
+                            <BaseFilters.Text>Collateral</BaseFilters.Text>
+                            <BaseFilters.Dropdown
+                                value={state.collateralFilter ?? 'All'}
+                                options={COLLATERAL_FILTER_OPTIONS}
+                                onSelect={onCollateralFilterSelect}
+                            />
+                        </BaseFilters.DropdownContainer>
+                        <BaseFilters.DropdownContainer>
+                            <TooltipSelector tooltip={{ key: TooltipKeys.PowerLeverage }}>
+                                <BaseFilters.Text>Power Leverage</BaseFilters.Text>
+                            </TooltipSelector>
+                            <BaseFilters.Dropdown
+                                value={state.leverageFilter}
+                                options={LEVERAGE_FILTER_OPTIONS}
+                                onSelect={onLeverageFilterSelect}
+                            />
+                        </BaseFilters.DropdownContainer>
+                    </BaseFilters.Wrapper>
+                    <BaseFilters.Wrapper>
+                        {!hideSideFilter && (
+                            <BaseFilters.DropdownContainer>
+                                <BaseFilters.Text>Side</BaseFilters.Text>
+                                <BaseFilters.Dropdown
+                                    value={state.sideFilter}
+                                    options={SIDE_OPTIONS}
+                                    onSelect={onSideFilterSelect}
+                                />
+                            </BaseFilters.DropdownContainer>
+                        )}
+                        <BaseFilters.DropdownContainer>
+                            <BaseFilters.Text>Sort</BaseFilters.Text>
+                            <BaseFilters.Dropdown
+                                value={state.sortBy}
+                                options={SORT_BY_OPTIONS}
```

Comment:
not sure about these sorting options, they seem a little weird for the staking page (they seem to just be re-used from the pools page)

![image](https://user-images.githubusercontent.com/14157626/168589076-bd48c4a7-d7f6-47d5-94d9-164a95201c3c.png)